capture_service/dive_client_cli.cc

```diff
@@ -560,103 +565,91 @@ absl::Status RetrieveGfxrCapture(Dive::DeviceManager& mgr, const GlobalOptions&
     return absl::OkStatus();
 }
 
-// Triggers a GFXR capture on the device, allowing for multiple captures and screenshot.
-absl::Status TriggerGfxrCapture(Dive::DeviceManager& mgr, const GlobalOptions& options)
+// Triggers a GFXR capture and screenshot on the device.
+absl::StatusOr<bool> TriggerGfxrCapture(Dive::DeviceManager& device_manager,
```

Comment:
Can this be made const?
```suggestion
absl::StatusOr<bool> TriggerGfxrCapture(const Dive::DeviceManager& device_manager,
```